lib/scss/jumpstart_ui.layout.scss

```diff
@@ -2,10 +2,10 @@
 
 // Jumpstart UI Specific Configuration and variable overrides.
 @import './jumpstart_ui.config';
-// @import 'decanter/core/src/scss/decanter-no-markup';
-//
-// // Render Decanter Library Base Styles Only.
-// @import 'decanter/core/src/scss/layout/index';
-//
-// // Jumpstart UI Custom Styles
-// @import './layouts/index';
+@import '~decanter/core/src/scss/decanter-no-markup';
+
+// Render Decanter Library Base Styles Only.
+@import '~decanter/core/src/scss/layout/index';
```

Comment:
Do you need these?

Reply:
Without `@import '~decanter/core/src/scss/decanter-no-markup';` I get an error (see below).
However, it appears that I don't need `@import '~decanter/core/src/scss/layout/index';`

Would it make better sense to move that into 'jumpstart_ui.config' ?

    ```
ERROR in ./lib/scss/jumpstart_ui.layout.scss
    Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
    ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
    SassError: Undefined variable.
      ╷
    5 │   @media #{$su-breakpoint-xl} {
      │            ^^^^^^^^^^^^^^^^^
      ╵

```